src/server.c

```diff
@@ -5468,11 +5460,10 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
     }
 
     /* Modules */
-    if (!server.sentinel_mode && (all_sections || (dictFind(section_dict,"modules") != NULL))) {
+    if (!server.sentinel_mode && (all_sections || (dictFind(section_dict,"module_list") != NULL))) {
```

Comment:
now that i think of it, maybe we wanna return this also if the `modules` section was requested.
i.e. this `if` will check both `modules` and `module_list`